frontend/src/app/features/work-packages/components/wp-share-modal/wp-share.modal.html

```diff
@@ -3,8 +3,18 @@
 >
   <div
     class="spot-modal--header"
-    [textContent]="this.text.title"
   >
+    <div
+      class="spot-modal--header-title"
+      [textContent]="this.text.title">
+    </div>
+    <svg
+      x-icon
+      class="spot-modal--header-close"
+      size="small"
+      (click)="closeMe($event)"
+      [attr.title]="text.button_close"
+    ></svg>
```

Comment:
I attempted to get to the "X" by tabbing and I'm not able to. We still have the accessible close button at the bottom which is reachable but I don't see how this one can be navigated to as it sets a negative tab index. Primer seems to accomplish this in their Dialog by rendering the SVG within an invisible-scheme button. I tried it out with:

```html
<div
    class="spot-modal--header"
  >
    <div
      class="spot-modal--header-title"
      [textContent]="this.text.title">
    </div>
    <div
      class="Overlay-actionWrap">
      <button
        [attr.aria-label]="text.button_close"
        type="button"
        class="close-button Overlay-closeButton"
        (click)="closeMe($event)">
        <svg
          x-icon
          size="small"
        ></svg>
      </button>
    </div>
  </div>
```
The only thing is that there's a weird double focus when clicking directly on the "X". It focuses the SVG and the Button itself. I couldn't figure out how to prevent focus on the SVG. I tried setting `[attr.focusable]="false"` on it but that didn't seem to do it.

If the navigational ability to this "X" is not a concern then ignore this comment :)

Reply:
I removed the doubled focus now with a CSS-rule. It is a bit hacky, but okay for me as it is only temporily anyways until we switch to the Primer Dialog

Reply:
Awesome. Looks identical to Primer's :)